src/django/pfb_network_connectivity/models.py

```diff
@@ -0,0 +1,21 @@
+"""
+Models related to analysis results.
+"""
+
+from __future__ import unicode_literals
+
+from django.db import models
+
+from base.models import PFBModel
+
+
+class Area(models.Model):
```

Comment:
Further to my brief comment below: Area, adapted from State, isn't really the same thing as Neighborhood, as far as I understand.  State was an attribute of `Organization` that served to link related organization entities together and had permissions implications.  Neighborhood is, I think, linked to a specific analysis or series of analysis but isn't something that's independent of an organization or has a useful meaning across organizations.

So I think if we want/need organization hierarchy we should go with something named to reflect what it's doing (like `Group`, or ideally something  a little more specific), and if we don't we can get rid of it.